src/software/jetson_nano/services/motor.cpp

```diff
@@ -562,6 +590,17 @@ void MotorService::configureEncoder(uint8_t motor)
     writeToControllerOrDieTrying(motor, TMC4671_ABN_DECODER_PPR, 0x00001000);
 }
 
+void MotorService::configureHall(uint8_t motor)
+{
+    // Digital hall settings
+    tmc4671_writeInt(motor, TMC4671_HALL_MODE, 0x00000000);
```

Comment:
why do we not need to write "or die trying" like with the dribbler/driver

Reply:
good catch, fixed!